slack/client.go

```diff
@@ -414,6 +415,52 @@ func (c *slackClient) loadFromStorage() error {
 	return nil
 }
 
+// Broadcast sends message, and returns the number of
+// processed bots, conversations, errors and error occurred
+func (c *slackClient) Broadcast(msg interface{}, cond flamingo.Condition) (uint64, uint64, uint64, error) {
+	var err error
+	conversationsCount := uint64(0)
+	clientBotsCount := uint64(0)
+	errorsFound := uint64(0)
+
+	for _, clientBot := range c.bots {
```

Comment:
Ok, so I saw this is not for error handing but for restriction purposes. Then I prefer something like this:

```go
type BroadcastFilter func(bot string, channel string) bool
func (c *Client) Broadcast(msg interface{}, filter BroadcastFilter) (stuff)
```